examples/index.html

```diff
@@ -335,7 +335,18 @@ <h4><b>By entering co-ordinates :</b></h4>
     </div>
 
     <script>
-     
+      
+      function myFunctionLocation() {
```

Comment:
`myFunctionLocation` is not an appropriate name for this function. `locateUserHandler` would be better.